src/components/cards/BiobankReportCard.vue

```diff
@@ -4,38 +4,65 @@
-        <h3>{{biobank.data.name}}</h3>
-
-        <small>
-          <dl class="row">
-            <dt class="col-sm-2"><b>Collection types:</b></dt>
-            <dd class="col-sm-10"><i>{{ collectionTypes }}</i></dd>
-            <dt class="col-sm-2"><b>Number of collections:</b></dt>
-            <dd class="col-sm-10"><i>{{ biobank.data.collections.length }}</i></dd>
-            <dt class="col-sm-2"><b>Number of samples:</b></dt>
-            <dd class="col-sm-10"><i>{{ numberOfSamples }}</i></dd>
-          </dl>
-        </small>
-
-        <p>{{biobank.data.description}}</p>
-
-        <br/>
-
         <div class="row">
-          <div class="col-md-6">
-            <ul class="list-unstyled">
-              <li><b>{{ biobank.data.juridical_person }}</b></li>
-              <li v-if="biobank.data.contact.address">{{ biobank.data.contact.address }}</li>
-              <li>{{ biobank.data.contact.zip }} {{ biobank.data.contact.city }}</li>
-              <li>{{ biobank.data.country.name }}</li>
-            </ul>
+          <div class="col-md-10">
+            <h3>{{biobank.data.name}}</h3>
+
+            <small>
+              <dl class="row">
+                <dt class="col-sm-3"><b>Collection types:</b></dt>
+                <dd class="col-sm-9"><i>{{ collectionTypes }}</i></dd>
+                <dt class="col-sm-3"><b>Number of collections:</b></dt>
+                <dd class="col-sm-9"><i>{{ biobank.data.collections.length }}</i></dd>
+                <dt class="col-sm-3"><b>Number of samples:</b></dt>
+                <dd class="col-sm-9"><i>{{ numberOfSamples }}</i></dd>
+              </dl>
+            </small>
+
+
+            <p>{{biobank.data.description}}</p>
+
+            <br/>
+
+            <div class="row">
+              <div class="col-md-6">
+                <ul class="list-unstyled">
+                  <li><b>{{ biobank.data.juridical_person }}</b></li>
+                  <li v-if="biobank.data.contact.address">{{ biobank.data.contact.address }}</li>
+                  <li>{{ biobank.data.contact.zip }} {{ biobank.data.contact.city }}</li>
+                  <li>{{ biobank.data.country.name }}</li>
+                </ul>
+              </div>
+              <div class="col-md-4">
+                <ul class="list-unstyled">
+                  <li><b>{{ biobank.data.contact.first_name }} {{ biobank.data.contact.last_name }}</b></li>
+                  <li>Email: {{ biobank.data.contact.email }}</li>
+                  <li>Tel: {{ biobank.data.contact.phone }}</li>
+                </ul>
+              </div>
+            </div>
           </div>
-          <div class="col-md-4">
-            <ul class="list-unstyled">
-              <li><b>{{ biobank.data.contact.first_name }} {{ biobank.data.contact.last_name }}</b></li>
-              <li>Email: {{ biobank.data.contact.email }}</li>
-              <li>Tel: {{ biobank.data.contact.phone }}</li>
-            </ul>
+          <div class="col-2">
```

Comment:
The bigger this card gets, the more attractive it becomes to split off some subcomponents